docs/question/template.md

```diff
@@ -1,6 +1,116 @@
+
+### `"question"` panel
+
+Text in `question.html` can be set to only display in the "question" panel by wrapping it in the [`<pl-question-panel>` element](../elements.md#pl-question-panel-element). This is useful for the question prompt, which doesn't need to be repeated in the "submission" and "answer" panels. For example:
+
+=== "Good"
+
+    ```html
+    <pl-question-panel>
+      <p>What is y, if y is 1 + 3?</p>
+    </pl-question-panel>
+    <pl-number-input answers-name="y" label="$y =$"></pl-number-input>
+    ```
+
+=== "Bad"
+
+    ```html
+    <p>What is y, if y is 1 + 3?</p>
+    <pl-number-input answers-name="y" label="$y =$"></pl-number-input>
+    ```
+
+    The question will be repeated in the "submission" and "answer" panels, which is unnecessary and can be confusing for students.
+
+### `"submission"` panel
+
+By default, feedback given by elements (`data["partial_scores"][NAME]["feedback"]`) is rendered when an element renders in the "submission" panel. You can use the [`pl-submission-panel`](../elements.md#pl-submission-panel-element) to show additional text only in the "submission" panel. For example:
+
+=== "Good"
+
+    ```html
+    <pl-submission-panel>
+      <p>
+        Feedback: {{feedback.y}}
+      </p>
+    </pl-submission-panel>
+    ```
+
+=== "Bad"
+
+    ```html
+    <p>Feedback: {{feedback.y}}</p>
+    ```
+
+    This will show the feedback in the "question" and "answer" panels as well, which is unnecessary and can be confusing for students. Since `feedback.y` won't be defined until the student submits an answer, it will show as an empty string in the "question" panel, just showing `"Feedback: "`.
+
+More details about providing feedback can be found on the [`server.py` page](server.md#providing-feedback).
+
+### `"answer"` panel
+
+By default, the correct answers are shown in the "answer" panel. This can be disabled by setting `showCorrectAnswer: false` in the question's `info.json`. The FAQ item ["How can I hide the correct answer when students see their grading results?"](../faq.md#how-can-i-hide-the-correct-answer-when-students-see-their-grading-results) describes some other techniques for hiding the correct answer.
+
+If you grade a question using a [custom grading function](./server.md#step-5-grade) or an [external grader](../externalGrading.md), it is recommended to set the correct answer so that students can see it in the answer panel. If there are multiple correct answers, you should add a note within the [`pl-answer-panel`](../elements.md#pl-answer-panel-element) element that any correct answer would be accepted, and the displayed answer is only an example. You can also show additional text in the answer panel as well, e.g. extended explanations for the correct answers.
```

Comment:
```suggestion
If you grade a question using a [custom grading function](./server.md#step-5-grade), an [external grader](../externalGrading.md), or using [manual grading](../manualGrading/index.md), you are encouraged to display a correct answer so that students can see it in the answer panel. If there are multiple correct answers, you should add a note within the [`pl-answer-panel`](../elements.md#pl-answer-panel-element) element that any correct answer would be accepted, and the displayed answer is only an example. You can also show additional text in the answer panel as well, e.g. extended explanations for the correct answers.
```

Reply:
Added in 6408ede3ac9fe51ab8d193ea476887f9d181752a.